5 sec loading message

realcodywburns opened the pull request:
While the page is loading in the background:
randomly selects 1 of 5 messages from from the translation file WELCOME_msg# 
Displays the actual url the user is connecting to: I.e. http://myetherwallet.com or http://scumbagsRus.com

tayvano committed:
hrefs don't go on buttons silly choose

realcodywburns committed:
fix loading msg

randomly selects 1 of 5 messages loaded from WELCOME_msg# in the translation file

crptm commented on app/scripts/controllers/genericCtrl.js:
```diff
@@ -14,9 +15,13 @@ var genericCtrl = function($scope, globalService, $sce) {
-        website : window.location.href
-      };
+    var randNumber = Math.floor(Math.random() * 5) + 1;
+    console.log( randNumber);
```
ohai

crptm replied:
msgs =['a','b','c']

return msgs[Math.floor(Math.random() * msgs.length)] is more straightforward way IMO

crptm replied:
Also given chosen message ids, we can just do 'WELCOM_msg' + randNumber

realcodywburns replied:
I agree much cleaner changing to 

$scope.loadingMsg = {
      msg : 'WELCOME_msg'+randNumber ,
      website : window.location.href
    }

tayvano replied:
~Sorry can we have a random message pulled from an array. I trying to make a warning one and I kinda made a lot. 😬~

Sorry I'm dumb. Below is all the tips. Will filter down later.

https://gist.github.com/tayvano/83e72b0f52f26293353bed194f140460

realcodywburns replied:
yep adding now. add as many as you want it will count them and pick a random one.
 mew-tips.js >> /scripts

realcodywburns committed:
Update genericCtrl.js

realcodywburns committed:
clean up randnumber msg

realcodywburns committed:
Merge branch 'x3.10.x-offline-singing-DP' of https://github.com/realcodywburns/etherwallet into x3.10.x-offline-signing-DP

realcodywburns committed:
mew-tip inclusion

picks a rand mew tip from mew-tips ,
add as many as you like
delay set to 8 secs
needs some css work

crptm commented on app/scripts/controllers/genericCtrl.js:
```diff
@@ -4,19 +4,22 @@ var genericCtrl = function($scope, globalService, $sce) {
-   }, 5000);
+   }, 8000);
 
+   console.log($scope.mewtip);
```
console.log

crptm commented on app/scripts/controllers/genericCtrl.js:
```diff
@@ -4,19 +4,22 @@ var genericCtrl = function($scope, globalService, $sce) {
+   }, 8000);
 
+   console.log($scope.mewtip);
+    var randNumber = Math.floor(Math.random() * $scope.mewtip.length) + 1;
```
Arrays are 0 indexed, +1 would never produce 0 index

realcodywburns committed:
fix randNumber length

rm +1 for 0 index array. allows mewtip[0] and will not chose mew tip out of array
rm console.log of mewtip also

tayvano commented:
I dunno what the fuck I did but I fucked it. I think it should actually be merged but not recognizing :( :( :(

tayvano commented:
Okay here's my launch list. I haven't slept in like 30 hours so its a mess. I'll circle back tonight, but at least its in writing so we can conquer. 

I have no idea whats going on with the branch. I know that I pulled in your branch - or at least I thought I did. it's definitely got the new random stuff now. 😭

https://gist.github.com/tayvano/b66fd9c327a9498573249c071f041d46

realcodywburns committed:
cleanup

lgtm

realcodywburns committed:
cleanup

lgtm

realcodywburns committed:
Merge branch 'x3.10.x-offline-singing-DP' of https://github.com/realcodywburns/etherwallet into x3.10.x-offline-singing-DP

crptm commented on app/scripts/translations/en.js:
```diff
@@ -317,7 +317,15 @@ z_TRANSLATE3                : ' ',
+
+/* welcome messages */
+
+WELCOME_msg1                : 'welcome msg 1',
```
I suppose these no longer needed

realcodywburns committed:
modified:   README.md
	modified:   app/includes/browser_action/browser_action.html
	modified:   app/includes/browser_action/cx-quicksend.html
	modified:   app/includes/bulkGenerate.tpl
	modified:   app/includes/contracts-deploy-1.tpl
	modified:   app/includes/contracts-interact-2.tpl
	modified:   app/includes/contracts-interact-modal.tpl
	modified:   app/includes/contracts.tpl
	modified:   app/includes/ens.tpl
	modified:   app/includes/footer.tpl
	modified:   app/includes/header.tpl
	modified:   app/includes/help.tpl
	modified:   app/includes/signMsg.tpl
	modified:   app/includes/swap-stage-1.tpl
	modified:   app/includes/swap-stage-2.tpl
	modified:   app/includes/swap-stage-3.tpl
	modified:   app/includes/txBroadcast-modal.tpl
	modified:   app/includes/txBroadcast.tpl
	modified:   app/includes/txSend-content.tpl
	new file:   app/includes/txSend-content.tpl~mercury
	modified:   app/includes/txSend-modal.tpl
	new file:   app/includes/txSend-modal.tpl~mercury
	modified:   app/includes/txSend.tpl
	modified:   app/includes/txView.tpl
	new file:   app/includes/txView.tpl~mercury
	modified:   app/includes/walletView.tpl
	new file:   app/includes/walletView.tpl~mercury
	modified:   app/layouts/cx-wallet.html
	modified:   app/layouts/embedded.html
	modified:   app/layouts/helpers.html
	modified:   app/layouts/index.html
	modified:   app/layouts/signmsg.html
	modified:   app/scripts/bumb.js
	modified:   app/scripts/controllers/contractsCtrl.js
	modified:   app/scripts/controllers/decryptWalletCtrl.js
	modified:   app/scripts/controllers/ensCtrl.js
	modified:   app/scripts/controllers/footerCtrl.js
	modified:   app/scripts/controllers/genericCtrl.js
	modified:   app/scripts/controllers/headerCtrl.js
	new file:   app/scripts/controllers/headerCtrl.js~mercury
	modified:   app/scripts/controllers/swapCtrl.js
	modified:   app/scripts/controllers/txBroadcastCtrl.js
	modified:   app/scripts/controllers/txSendCtrl.js
	modified:   app/scripts/controllers/txSignCtrl.js
	modified:   app/scripts/controllers/walletDecryptOfflineCtrl.js
	modified:   app/scripts/controllers/walletViewCtrl.js
	new file:   app/scripts/controllers/walletViewCtrl.js~mercury
	modified:   app/scripts/customGas.js
	modified:   app/scripts/cxFuncs.js
	modified:   app/scripts/directives/addressFieldDrtv.js
	modified:   app/scripts/directives/balanceDrtv.html
	modified:   app/scripts/directives/notwalletDecryptOfflineDrtv.html
	modified:   app/scripts/directives/walletDecryptDrtv.html
	modified:   app/scripts/directives/walletDecryptOfflineDrtv.html
	modified:   app/scripts/directives/walletDecryptOfflineDrtv.js
	modified:   app/scripts/ethFuncs.js
	modified:   app/scripts/main.js
	modified:   app/scripts/mew-tips.js
	modified:   app/scripts/services/globalService.js
	modified:   app/scripts/services/walletService.js
	modified:   app/scripts/translations/ar.js
	modified:   app/scripts/translations/de.js
	modified:   app/scripts/translations/el.js
	modified:   app/scripts/translations/en.js
	modified:   app/scripts/translations/es.js
	modified:   app/scripts/translations/fi.js
	modified:   app/scripts/translations/fr.js
	modified:   app/scripts/translations/ht.js
	modified:   app/scripts/translations/hu.js
	modified:   app/scripts/translations/id.js
	modified:   app/scripts/translations/it.js
	modified:   app/scripts/translations/ja.js
	modified:   app/scripts/translations/ko.js
	modified:   app/scripts/translations/nl.js
	modified:   app/scripts/translations/no.js
	modified:   app/scripts/translations/pl.js
	modified:   app/scripts/translations/pt.js
	modified:   app/scripts/translations/ru.js
	modified:   app/scripts/translations/tr.js
	modified:   app/scripts/translations/vi.js
	modified:   app/scripts/translations/zhcn.js
	modified:   app/scripts/translations/zhtw.js
	modified:   app/scripts/uiFuncs.js
	modified:   app/styles/etherwallet-custom.less
	modified:   app/styles/etherwallet-ext-custom.less
	modified:   app/styles/etherwallet-master.less
	modified:   app/styles/etherwallet-nav.less
	modified:   app/styles/etherwallet-new.less
	modified:   app/styles/etherwallet-variables.less
	modified:   app/tests/index.html
	modified:   chrome-extension/browser_action/browser_action.html
	modified:   chrome-extension/browser_action/cx-quicksend.html
	modified:   chrome-extension/css/etherwallet-master.min.css
	modified:   chrome-extension/cx-wallet.html
	modified:   chrome-extension/helpers.html
	modified:   dist/README.md
	modified:   dist/bin/mew-tips.js
	modified:   dist/css/etherwallet-master.min.css
	modified:   dist/embedded.html
	modified:   dist/helpers.html
	modified:   dist/index.html
	modified:   dist/signmsg.html

realcodywburns committed:
modified:   chrome-extension/css/etherwallet-master.min.css
	modified:   chrome-extension/cx-wallet.html
	new file:   chrome-extension/embedded.html
	modified:   chrome-extension/helpers.html
	new file:   chrome-extension/images/fav/manifest.json
	new file:   chrome-extension/index.html
	modified:   chrome-extension/js/etherwallet-master.js
	new file:   chrome-extension/package.json
	new file:   chrome-extension/signmsg.html
	modified:   dist/css/etherwallet-master.min.css
	new file:   dist/cx-wallet.html
	modified:   dist/embedded.html
	modified:   dist/helpers.html
	new file:   dist/images/icons/icon128.png
	new file:   dist/images/icons/icon16.png
	new file:   dist/images/icons/icon19.png
	new file:   dist/images/icons/icon48.png
	modified:   dist/index.html
	modified:   dist/js/etherwallet-master.js
	new file:   dist/manifest.json
	modified:   dist/signmsg.html

realcodywburns commented:
closing, migrated to #732 for version 3.10
